project/Dependencies.scala

```diff
@@ -8,7 +8,8 @@ object Dependencies {
   lazy val spark = "org.apache.spark" %% "spark-core" % "2.2.0"
   lazy val fixNetty = "io.netty" % "netty-all" % "4.1.17.Final"
   lazy val fixNewerHadoopClient = "org.apache.hadoop" % "hadoop-client" % "2.7.2"
-  lazy val engine = "tech.sourced" % "engine" % "0.5.7" classifier "slim"
+  lazy val engine = ("tech.sourced" % "engine" % "0.5.7" classifier "slim")
+    .exclude("com.google.shadedcommon", "util")
```

Comment:
Could you please help to understand, why is this exclusion needed?

In general, any library can have things shaded that it needs (of particular version, etc) - and it's implementation detail.  Shading would guarantee class names are uniq and only used inside the library itself (so there should be no collisions).

Reply:
sorry. leftover. that's why PR is WIP :)
rename of rename is done.